app/views/coronavirus/reorder_timeline_entries/_reorder_list.html.erb

```diff
@@ -1,6 +1,12 @@
 <%
   timeline_entries = @page.timeline_entries.order(:position)
-  timeline_entries = timeline_entries.map { |te| { id: te.id, title: te.heading } }
+  timeline_entries = timeline_entries.map do |te|
+    {
+      id: te.id,
+      title: te.heading,
+      description: te.national_applicability.join(", ").titleize
```

Comment:
Do you fancy implementing "UK Wide" in this?

I imagine we could have `TimelineEntry#applicability_text` which had an implementation of:

```ruby
if te.national_applicability.uniq.sort == UK_NATIONS
  "UK Wide"
else
  national_applicability.join(", ").titleize
end
```

Reply:
👍 Done and fixed up.